development/tests.py

```diff
@@ -42,3 +50,34 @@ def test_create_match(
         self.assertEqual(match.bot_2, bot_two)
         self.assertEqual(match.score_p_1, scr1)
         self.assertEqual(match.score_p_2, scr2)
+
+
+class TestForms(TestCase):
+    def test_get_online_bots(self):
+        users = ['Pedro', 'Pablo', 'Gabi']
+        self.assertEqual(
+            list(get_online_bots(users)),
+            [
+                (0, 'Pedro'),
+                (1, 'Pablo'),
+                (2, 'Gabi'),
+            ],
+        )
+
+    def test_get_my_bots(self):
+        user = User.objects.create(
+            email='email',
+            username='username'
+        )
+        Bot.objects.create(
+            name='bot_name',
+            token='asd123',
+            user=user,
+        )
+        online_bots = ['Pedro', 'bot_name', 'Pablo']
+        self.assertEqual(
+            list(get_my_bots(user, online_bots)),
+            [
+                (0, 'bot_name')
+            ],
+        )
```

Comment:
You are not testing forms here. Actually, you're testing helper methods. Also, they are quite different, could you split them into different test suites?

Reply:
for the next sprint!